### ISSUE TITLE
Accessibility considerations for standards / grammar

### ISSUE THREAD

**Issue by agvbergin** [Wardley-Mapping-Foundation/wmf-standards-dsl#10]
Defining machine-readable standard grammar for Wardley mapping brings with it an opportunity to increase accessibility of Wardley mapping outside of purely visual representations of maps, e.g. for blind and low vision screen reader users and those who primarily navigate via keyboard.

An interesting feature of some of the existing grammars for Wardley mapping is that they leverage the visual look of the characters, e.g. Online Wardley mapping connects components using a hyphen and a greater than sign`->` to make what looks in composite like an arrow, and in Wardley To Go dots and vertical lines are used to suggest an axis`evolution: |....|....|..x..|........|`.

These representations may cause issues for screen reader users, particularly as screen readers have specific settings for which punctuation characters they announce. VoiceOver on macOS with default settings for example announces `evolution: |....|....|..x..|........|` as "evolution x" which doesn't meet POUR criteria.

We will need to solicit feedback from blind screen reader users in our community on these aspects of the grammars to ensure that they have a good experience in reading and writing Wardley maps.
